db/migrate/20200811181056_reset_fingerprint_column.rb

```diff
@@ -0,0 +1,15 @@
+require 'rake'
+
+# Upgrading to FIPs complaint hashing caused us to update the fingerprint
+# hash to utilize SHA256 instead of MD5: (https://github.com/cyberark/slosilo/pull/15).
+# The following rake task recalculates the fingerprint hashes
+
+Sequel.migration do
+  up do
+    Rake::Task['slosilo:recalculate_fingerprints'].execute
+  end
+
+  down do
+    raise("Downgrading is not supported due to underlying OpenSSL issue.")
+  end
```

Comment:
Looking at https://sequel.jeremyevans.net/rdoc/files/doc/migration_rdoc.html, the right thing to do here is leave out the `down` block if there is no downgrade step that correspond to the upgrade.